Source/Pages/Gallery/YPLibraryVC+PanGesture.swift

```diff
@@ -9,159 +9,233 @@
 import UIKit
 
 public class PanGestureHelper: NSObject, UIGestureRecognizerDelegate {
-    
-    var v: YPLibraryView!
+
+    var v: YPLibraryViewPanning!
```

Comment:
nit: I know this naming existed before this change but since we are making changes here what do you think about updating the variable name to be more descriptive and not just a single letter variable name? 😄

Reply:
Yeah the original naming matched the original naming conventions in the library. I renamed it now.